app/daemon/daemonapi/wallet.go

```diff
@@ -30,22 +32,34 @@ func checkPasswordStrength(pass string) error {
 
 }
 
-// encryptWallet executes json RPC command and returns...
-// func encryptWallet(w http.ResponseWriter, r *http.Request) {
+// encryptWallet executes json RPC command and returns response
+func encryptWallet(w http.ResponseWriter, r *http.Request) {
+
+	// temp valid password until we have UI setup
+	validPass := "d1924ce3d0510b2b2b4604c99453e2e1"
```

Comment:
This should use the password passed in the post

Reply:
ok... yeah was just hard-coding for now until we have ui... but can add it to the test route